packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [changed] Network streams are automatically closed after 60 seconds of
+  idleness.
+- [changed] We no longer log 'RPC failed' messages for expected failures.
 
 # v0.1.2
 - [fixed] Fixed validation of nested arrays to allow indirect nesting
```

Comment:
- [changed] We now support field deletes for set() with `{merge:true}`.

Reply:
Added:
- [changed] We now support `FieldValue.delete()` sentinels in `set()` calls
  with `{merge:true}`.